data/impact.js

```diff
@@ -0,0 +1,98 @@
+import sampleImpactImage from '../public/images/impact/sample-image.png';
+
+export const initiatives = [
+	{
+		title: 'A workshop 1',
+		img: sampleImpactImage,
+		alt: 'sample image - will change later, I promise!',
+		description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.',
+	},
+	{
+		title: 'A workshop 2',
+		img: sampleImpactImage,
+		alt: 'sample image - will change later, I promise!',
+		description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.',
+	},
+	{
+		title: 'A workshop 3',
+		img: sampleImpactImage,
+		alt: 'sample image - will change later, I promise!',
+		description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.',
+	},
+];
+
+export const people = [
+	{
+		name: 'Maya Raman',
+		pronouns: 'she/her',
+		position: 'Impact Director',
+		major: 'Computer Science Major',
+		year: 2023,
+		img: 'https://teachla.uclaacm.com/img/team/mayaraman.jpg',
+		alt: 'a photo of maya smiling beside royce hall',
+	},
+	{
+		name: 'Maya clone #1',
+		pronouns: 'she/her',
+		position: 'Impact Director',
+		major: 'Computer Science Major',
+		year: 2023,
+		img: 'https://teachla.uclaacm.com/img/team/mayaraman.jpg',
+		alt: 'a photo of maya smiling beside royce hall',
+	},
+	{
+		name: 'Maya clone #2',
+		pronouns: 'she/her',
+		position: 'Impact Director',
+		major: 'Computer Science Major',
+		year: 2023,
+		img: 'https://teachla.uclaacm.com/img/team/mayaraman.jpg',
+		alt: 'a photo of maya smiling beside royce hall',
+	},
+];
+
+const blogImages = [
+	'https://miro.medium.com/max/1400/0*CtF08rCYE9G4Xqk7.jpg',
+	'https://miro.medium.com/max/1400/0*LSHF7HtyCOuJgbBP.jpg',
+	'https://cdn.shopify.com/s/files/1/0070/7032/files/how-to-start-a-blog-illustration.png?format=jpg&quality=90&v=1595363254&width=1024',
+];
```

Comment:
A bit confused about how the `blogImage` array differs from the `img` filed of `blogs` - could we instead just construct `backgroundStyle` using what's already in `img`, and get rid of this variable?